extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -20,9 +20,9 @@ const makeDisplaySet = instances => {
     SeriesTime: instance.SeriesTime,
     SeriesInstanceUID: instance.SeriesInstanceUID,
     StudyInstanceUID: instance.StudyInstanceUID,
-    SeriesNumber: instance.SeriesNumber,
+    SeriesNumber: instance.SeriesNumber || 0,
     FrameRate: instance.FrameTime,
-    SeriesDescription: instance.SeriesDescription,
+    SeriesDescription: instance.SeriesDescription || 'Unspecified',
```

Comment:
probably needs a translation string

Reply:
What about just using an empty string?  That is what is supposed to come back?  I'm going to change it to empty.